src/java.base/share/classes/java/security/PEM.java

```diff
@@ -105,16 +122,28 @@ public record PEMRecord(String type, String content, byte[] leadingData)
      * @throws NullPointerException if {@code type} and/or {@code content} are
      * {@code null}.
      */
-    public PEMRecord(String type, String content) {
+    public PEM(String type, String content) {
         this(type, content, null);
     }
 
     /**
-     * Returns the type and Base64 encoding in PEM format.  {@code leadingData}
-     * is not returned by this method.
+     * Returns the type and Base64 encoding in PEM textual format.
+     * {@code leadingData} is not returned by this method.
      */
     @Override
     public String toString() {
         return Pem.pemEncoded(this);
     }
+
+    /**
+     * Returns a Base64 decoded byte array of {@code content}.
```

Comment:
Suggest stating that each invocation of this method returns a new byte array, as it isn't clear.